efel/tests/test_cppcore.py

```diff
@@ -154,3 +154,22 @@ def test_getFeature_non_existant(self):
         """cppcore: Testing failure exit code in getFeature"""
         import efel.cppcore
         efel.cppcore.getFeature("does_not_exist", list())
+
+    def test_registerFeature(self):
+        import efel.cppcore
+
+        def getTwiceAP_Amp(output):
+            feature_values = list()
+            efel.cppcore.getFeature('AP_amplitude', feature_values)
+            output.extend(2*x for x in feature_values)
+            return len(feature_values)
+
+        self.setup_data()
+        efel.cppcore.registerFeature('getTwiceAP_Amp', getTwiceAP_Amp)
```

Comment:
What about just using the name of the feature do it: 'AP_amplitude_v1', 'AP_amplitude_v2', etc.  That way, the calling convention doesn't have to change.  Otherwise, getFeature has to take a version as well.